### DISCUSSION TITLE
farm cant see plots

### DISCUSSION THREAD

**Discussion by ditch73** [Chia-Network/chia-blockchain#6833] [Support]
My farm see hdd but can't see plots in it after restart computer. Could Core-pool cause this problem. Farm cant see new harvested plots too. I have farmed more then 200 plots for mounth and now my nod cant see any of them.

**Comment by XIII39** [comment]
For the plots on your farmer, ensure that the plot directories are configured. If you're on the GUI, click the plot tab, click add plot directories and the list of your plot directories will populate. If any directories are missing, add them. You can also open the `config.yaml` file (Windows: C:\Users\YOUR_USER_NAME\.chia\mainnet\config\config.yaml - Linux: ~/.chia/mainnet/config/config.yaml) and add them there.

As for your harvester machines, you need to add the directories with the plots in the `config.yaml` file on your harvester machine and be sure to point your harvester to your farmers local IP address.

The plots on your harvester will not show in the total plots on the farmer, you'll see the harvesters responding to challenges though. You can always look at the log file (/.chia/mainnet/log/debug.log) and search for you local IP, here you will see the farmer sending challenges to the harvester and the harvester responding to them

**Comment by ditch73** [comment]
Many thanks for help. I follow you hints but couldn't sort out how to add plots to config.yaml. Anyway I have changed existing config.yaml and install new one and its work. All my plots back on business. Can you tell me if I am in Core-pool do my plots can still win XCH in Chia NOD.

**Comment by BrandtH22** [comment]
@ditch73 , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
